<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputReplyWindow</class>
 <widget class="QMainWindow" name="inputReplyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>试卷读入</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonReply">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择试卷</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditReply">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>751</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBarInputReply"/>
 </widget>
 <resources/>
 <connections/>
</ui>
